docs/_posts/2016-11-03-ace3-version381.md

```diff
@@ -0,0 +1,60 @@
+
+Two new features have made it into 3.8.0 which are "blood splatters" and "ammo box cook off".
+Wounded units will now leave more visible trails of blood on the ground which could make for interesting missions like escaping as a wounded soldier who is being tracked down by enemy combatants.
+When ammo boxes are destroyed the contained ammunition now will cook off in a multitude of small explosions and the boxes won't just sink into the ground.
+
+<div class="row">
+    <div class="small-12 columns">
+        <img src="{{site.baseUrl}}/img/news/161003_blood.jpg" alt="A wounded insurgent in front of a damaged truck."/>
+    </div>
+</div>
+
+With 3.8.0 the map tools will allow you to draw straight lines again. A feature that sadly is missing in vanilla Arma 3.
+
+## The Future things
+
+### Medical rewrite
+
+If you're following ACE3 development on github closely you might already have spotted it: The medical system gets a huge rewrite. That's true, we're completely redoing the medical system. Until now the medical system has been a merge of the AGM and CSE medical systems. The AGM one kind of became what you know as the "basic system" and CSE's respectively the "advanced system". Being two separate systems it was always hard for us to improve one or the other without reworking the other one too. Not an ideal situation. Another issue we will address with this rewrite is the amount of influence a medical condition has on gameplay. An issue we've often heard from different communities as well. There's an [interesting discussion going on github with a lot of different ideas and feature requests](https://github.com/acemod/ACE3/issues/3134{:target="_blank"}). Worth a read.
+
+With the state machine system that has been introduced in CBA we are now able to strictly define various health states of players such as unconsciousness or cardiac arrest and what can happen in those states. This allows us to massively declutter the code which will not only reduce the possibility of bugs, but also enable us to add more features and gameplay elements to the medical system - with no performance loss at all.
+
+We're really looking forward to the result and how you will like it.
+
+### Night Vision Googles Improvements
+
+If you've [watched dslyecxi's recent youtube videos](https://www.youtube.com/user/Dslyecxi){:target="_blank"} or if you're [following us on twitter](https://twitter.com/intent/follow?screen_name=ace3mod&tw_p=followbutton){:target="_blank"} you might already have seen that dslyecxi has put together a mod vastly improving night vision googles and the resulting very limiting view. Check out his video with the explanation why vanilla Arma 3 <abbr title="Night Vision Goggles">NVG</abbr>s are a bad representation of real life NVGs and what he has done to improve that.
```

Comment:
- `youtube` -> `YouTube`
- "a bad representation" -> maybe reword it to "an inaccurate representation"